test/remote/sms_tests.js

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const assert = require('insist')
+const TestServer = require('../test_server')
+const Client = require('../client')()
+const config = require('../../config').getProperties()
+
+describe('remote sms', () => {
+  let server
+
+  before(() => {
+    config.sms.enabled = true
+    config.sms.isStatusGeoEnabled = true
+
+    return TestServer.start(config)
+      .then(result => {
+        server = result
+      })
+  })
+
+  it('GET /sms/status', () => {
+    return Client.create(config.publicUrl, server.uniqueEmail(), 'wibble')
+      .then(client => {
+        return client.smsStatus()
+          .then(status => {
+            assert.ok(status)
+            assert.equal(typeof status.ok, 'boolean')
+            assert.equal(status.country, 'US')
```

Comment:
Does it work because of [this x-forwarded-for](https://github.com/mozilla/fxa-auth-server/pull/1766/files#diff-3d1fc553e7b733f694e2adf7dc17f8d7R601) header up there?